notebooks/community/ml_ops/stage2/get_started_bqml_training.ipynb

```diff
@@ -38,6 +38,11 @@
         "      View on GitHub\n",
```

Comment:
<p>You should be able to move deleting the temporary BQ table out of the delete_storage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
Done !!